### PR TITLE
add arabic translation

### PR DESCRIPTION
**Pull request by painor**
This is the Arabic translation for the main website. 
the URL would look like http://dontasktoask.com/ar

### CODE REVIEW

**Comment by painor** [comment]
*caugh caugh*

File: ar/index.html [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+      Other similar problems: <a href="http://xyproblem.info/">The XY Problem</a>، <a href="http://nohello.com/">No Hello</a>.
+      Further reading: <a href="https://stackoverflow.com/help/how-to-ask">How do I ask a good question?</a>،
+      or if you have more time: <a href="http://catb.org/~esr/faqs/smart-questions.html">How To Ask Questions The Smart Way</a>.
+    </p>
+  </main>
+  <footer>
+    Content mostly stolen from <a href="http://iki.fi/sol/dontask.html">iki.fi/sol/dontask.html</a>
+    -
+    <a href="https://github.com/maunium/dontasktoask.com">Source on GitHub</a>
+  </footer>
```
Comment (tulir) [inline ar/index.html]: You can add translation credits in the footer

File: ar/index.html [hunk 2]
```diff
@@ -0,0 +1,164 @@
+
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Don't ask to ask، just ask</title>
+  <meta property="og:title" content="Don't ask to ask، just ask" />
```
Comment (tulir) [inline ar/index.html]: These should probably be translated as well?

File: ar/index.html [hunk 3]
```diff
@@ -0,0 +1,164 @@
+
```
Comment (tulir) [inline ar/index.html]:
```suggestion
```

File: ar/index.html (hunk 1 above)
Comment (painor) [reply ar/index.html]: do I add the telegram username or the github url ?

Comment (tulir) [reply ar/index.html]: whatever you want

**Comment by painor** [comment]
Everything should be fine now hopefully.

**Pull request merged**